lib/acl.c

```diff
@@ -271,7 +296,7 @@ init_ipv6_acls(struct gatekeeper_if *iface)
 
 	/* Add drop function for packets that have no match. */
```

Comment:
This comment should not be changed, since packets without a match will no longer be dropped.